app/navigation/stack-param-lists.ts

```diff
@@ -59,13 +60,13 @@ export type RootStackParamList = {
     domain: string
     k1: string
     defaultDescription: string
-    minWithdrawableSatoshis: number
-    maxWithdrawableSatoshis: number
+    minWithdrawableSatoshis: MoneyAmount<typeof WalletCurrency.Btc>
+    maxWithdrawableSatoshis: MoneyAmount<typeof WalletCurrency.Btc>
     receiveCurrency: WalletCurrency
-    walletId: string
-    satAmount: number
-    satAmountInUsd: number
-    amountCurrency: string
+    receivingWalletDescriptor: WalletDescriptor<typeof WalletCurrency.Btc>
```

Comment:
lnurlWithdraw is always connected to the BTC wallet?

Reply:
Until we can create usd invoices using sat amounts